auth/user_mgt.go

```diff
@@ -822,6 +834,7 @@ var serverError = map[string]string{
 	"DUPLICATE_LOCAL_ID":      uidAlreadyExists,
 	"EMAIL_EXISTS":            emailAlredyExists,
 	"INSUFFICIENT_PERMISSION": insufficientPermission,
+	"PERMISSION_DENIED":       insufficientPermission,
```

Comment:
Interesting.

Reply:
What I meant by that is that it seems a little odd for two different strings to map to the same error type. Is that intentional?

Reply:
We currently don't have a public permission denied error code: https://firebase.google.com/docs/auth/admin/errors

This will have to do for the moment. I'll run it by the auth team and see if we can get another public error code.

Reply:
Auth team is ok with this. We define `insufficient-permission` error code as follows:

```
The credential used to initialize the Admin SDK has insufficient permission to access the requested Authentication resource. See Add Firebase to your app for documentation on how to generate a credential with appropriate permissions and use it to authenticate the Admin SDKs.
```

Seems to fit the bill in this case.